### ISSUE TITLE
Noise in NSynth Dataset

### ISSUE THREAD

**Issue by stefan-balke** [magenta/magenta#2092]
Hi there,

I wanted to point out that in the NSynth dataset's test set, the following samples are noise.
* brass_acoustic_046-084-025.wav
* ...
* brass_acoustic_046-108-100.wav

So in total 71 items.
Haven't checked systematically but maybe there are more, even in the training set @jesseengel?
Happy to discuss further steps.
